src/redux/actions.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+export const FETCH_RANDOM_GREETING_SUCCESS = 'FETCH_RANDOM_GREETING_SUCCESS';
+
+export const fetchRandomGreetingSuccess = (greeting) => ({
+  type: FETCH_RANDOM_GREETING_SUCCESS,
+  payload: greeting,
+});
+
+export const fetchRandomGreeting = () => (dispatch) => {
+  axios.get('http://localhost:3000/random_greeting')
+    .then((response) => {
+      dispatch(fetchRandomGreetingSuccess(response.data.greeting));
+    })
+    .catch((error) => {
+      console.error('Error fetching random greeting:', error);
+    });
+};
```

Comment:
- You have done well so far, though after resolving the CORS error from the backend, your project works fine, in the microverse curriculum, it is better to use the `redux-toolkit` as this helps to prevent a to of boilerplate code that comes with using the `redux` library, making our code concise and easy to read/understand. I suggest you use the [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) function to make API calls. Please fix this 👍 👍